utils/test.go

```diff
@@ -0,0 +1,15 @@
+package utils
+
+import "testing"
+
+func CheckNum(num, expected int, t *testing.T) {
+	if num != expected {
+		t.Errorf("got %d, expected %d", num, expected)
+	}
+}
+
+func CheckStr(str, expected string, t *testing.T) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function CheckStr should have comment or be unexported